README.md

```diff
@@ -925,6 +925,59 @@ Back endpoint responsible for fetching and sending information to the database r
 `api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not: true = symptomatic, false = not symptomatic.
 
 
+### Jobs
+Back endpoint responsible for fetching and sending information to the database regarding Jobs and timesheet used to clock shifts.
+
+##### GET 
+
+`api/jobs/getJobs` Gets the users current active jobs and returns them in a list.  
```

Comment:
"get" should not be in the endpoint name;  it's already a GET.

Reply:
This needs "student" in the name, to distinguish from staff jobs (elsewhere).  Since it's getting all of them and it's in api/jobs, that's all that's necessary.